packages/ui/src/memberships/modals/TransferInviteModal/TransferInviteFormModal.tsx

```diff
@@ -17,15 +21,46 @@ interface Props {
   member?: Member
 }
 
+const formDefaultValues = {
+  from: undefined,
+  amount: undefined,
+  to: undefined,
+}
+
 export function TransferInviteFormModal({ onClose, onAccept, icon, member }: Props) {
-  const [from, setFrom] = useState<Member | undefined>(member)
-  const [to, setTo] = useState<Member>()
-  const [amount, setAmount] = useNumberInput(0)
-  const filterRecipient = useCallback(filterMember(from), [from])
+  const filterRecipient = useCallback(filterMember(member), [member])
+  const TransferInviteSchema = Yup.object().shape({
```

Comment:
It would be good to transfer schema out of component. Maximum invite count can be handled with context on useForm and maxContext helper for yup